arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -138,6 +138,9 @@ DECLARE_COUNTER(arangodb_revision_tree_resurrections_total, "Number of revision
 std::string const RocksDBEngine::EngineName("rocksdb");
 std::string const RocksDBEngine::FeatureName("RocksDBEngine");
 
+// global flag to cancel all compactions. will be flipped to true on shutdown
+std::atomic<bool> cancelCompactions{false};
```

Comment:
Can be static

Reply:
Yes. But it is already in the anynomous namespace and in the .cpp file... I think declaring it static in addition does not provide any benefit here, or does it?

Reply:
> anynomous namespace

should be I suppose

But I don't see this

Reply:
> I think declaring it static in addition does not provide any benefit here, or does it?

If we already have anynomous namespace I suppose no benefits

Reply:
Awww, my bad. The definition was in `namespace arangodb`, not the anonymous namespace.
But anyway, I changed it to be static in my last commit, so we should be fine now.